internal/process/process.go

```diff
@@ -227,6 +226,10 @@ func Load(configBlob []byte) (*config.Config, error) {
 			}
 			*peerData.PreExportFinal += "\n" + string(content)
 		}
+		if peerData.DefaultLocalPref != nil && peerData.OptimizeInbound != nil {
+			log.Warnf("Both DefaultLocalPref and OptimizeInbound are set," +
```

Comment:
Let's keep this on one line and clarify that the optimizer won't be able to optimize this peer. On second thought, this should be a fatal error instead of a warning.